draft-parecki-oauth-first-party-apps.md

```diff
@@ -867,6 +869,10 @@ These design decisions should enable authorization server implementations to iso
 
 # Document History
 
+-latest
```

Comment:
do you want to use `-latest` here? Typically I've seen the coming draft number used, like `-02`, but maybe I don't know about some convention or something...

Reply:
I'll update that before I publish